module/depinject.go

```diff
@@ -44,9 +40,10 @@ type ModuleInputs struct {
 	StoreService store.KVStoreService
 	AddressCodec address.Codec
 
-	StakingKeeper  stakingkeeper.Keeper
-	SlashingKeeper slashingkeeper.Keeper
-	BankKeeper     bankkeeper.Keeper
+	// TODO: Breaks depinject
+	//StakingKeeper  stakingkeeper.Keeper
```

Comment:
`depinject` doesn't like that we're using the keeper directly instead of an interface to it. 

@Reecepbcups I'll get back at it next week if you have any thoughts in the meantime...